### PR TITLE
fix: fix e2e unstable

### PR DESCRIPTION
**Pull request by LiuTianyou**
## What's changed?

1. fix e2e unstable

## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: collector/src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -94,6 +94,7 @@
     <!-- Production environment configuration -->
     <springProfile name="prod">
         <root level="INFO">
+            <appender-ref ref="ConsoleAppender"/>
```
Comment (Aias00) [inline collector/src/main/resources/logback-spring.xml:97]: remove this pls

File: manager/src/main/resources/logback-spring.xml [hunk 2]
```diff
@@ -94,6 +94,7 @@
     <!-- Production environment configuration -->
     <springProfile name="prod">
         <root level="INFO">
+            <appender-ref ref="ConsoleAppender"/>
```
Comment (Aias00) [inline manager/src/main/resources/logback-spring.xml:97]: remove this pls

**Review by Aias00 — CHANGES_REQUESTED** [review]

File: collector/src/main/resources/logback-spring.xml (hunk 1 above)
Comment (LiuTianyou) [reply collector/src/main/resources/logback-spring.xml:97]: hi，I will fix this later, I am trying to reproduce and fix the e2e instability issue

**Review by LinuxSuRen — COMMENTED** [review]
[api-testing](https://github.com/LinuxSuRen/api-testing/blob/b6188a0e6831b1af4df52ea63d509d298848ac14/e2e/test-suite-common.yaml#L18) has the similar way to check if the server is ready. But it cannot send a payload to the endpoint.

I'm wondering if there is an API for the health check. Return status code `200` when the server is ready to accept requests.

**Comment by LiuTianyou** [comment]
> [api-testing](https://github.com/LinuxSuRen/api-testing/blob/b6188a0e6831b1af4df52ea63d509d298848ac14/e2e/test-suite-common.yaml#L18) has the similar way to check if the server is ready. But it cannot send a payload to the endpoint.
> 
> I'm wondering if there is an API for the health check. Return status code when the server is ready to accept requests.`200`

@LinuxSuRen Hello, thanks for your reply. I used the login API of hertzbeat to check whether the service is ready, but it seems that the e2e instability is not caused by inaccurate health checks.We have encountered a situation where the API that requires authentication occasionally fails in the test case. Is there a debug mode in api-testing that can output more log information, such as the URL, header, payload of the sent http request, etc. to help us troubleshoot this problem?

**Comment by LinuxSuRen** [comment]
Sure. You can add the log output level flag, such as: `atest run --level debug`

**Comment by LiuTianyou** [comment]
> Sure. You can add the log output level flag, such as: `atest run --level debug`

@LinuxSuRen Thank you. got it

**Comment by yuluo-yx** [comment]
> > Sure. You can add the log output level flag, such as: `atest run --level debug`
> 
> @LinuxSuRen Thank you. got it

Current log message warn level

**Review by yuluo-yx — APPROVED** [review]
LGTM

**Comment by yuluo-yx** [comment]
@Aias00 if no other problem, maybe we can merge it, Let's see if the e2e test after that works.

**Comment by Aias00** [comment]
> @Aias00 if no other problem, maybe we can merge it, Let's see if the e2e test after that works.

sure

**Review by Aias00 — APPROVED** [review]

**Pull request merged**